src/main/java/tigase/xmpp/impl/BindResource.java

```diff
@@ -61,6 +67,9 @@
 	private static final Element[] DISCO_FEATURES = {new Element("feature", new String[]{"var"}, new String[]{XMLNS})};
 	private static final String RESOURCE_PREFIX_DEF = "tigase-";
 	private static int resGenerator = 0;
+	@Inject(nullAllowed = true)
+	private List<ClientConnectionManager> clientConnectionManagers;
```

Comment:
Please verify that those instances will actually be injected here. I think that they should be but with scopes in kernel that may not be true.

Reply:
You were right. I created separate bean to perform that check.

Reply:
It looks OK.